src/mlpack/methods/decision_tree/decision_tree.hpp

```diff
@@ -15,20 +15,9 @@
 
 #include <mlpack/core.hpp>
 
-#include "gini_gain.hpp"
-#include "information_gain.hpp"
-#include "mad_gain.hpp"
-#include "mse_gain.hpp"
-
-#include "best_binary_numeric_split.hpp"
-#include "random_binary_numeric_split.hpp"
-
-#include "best_binary_categorical_split.hpp"
-#include "all_categorical_split.hpp"
-
-#include "all_dimension_select.hpp"
-#include "random_dimension_select.hpp"
-#include "multiple_random_dimension_select.hpp"
+#include "gain_functions/gain_functions.hpp"
```

Comment:
Since this is called `FitnessFunction` in [the documentation](https://www.mlpack.org/doc/user/methods/decision_tree.html#advanced-functionality-template-parameters), it probably makes sense to use the directory name `fitness_functions/` here.